apiserver/facades/controller/usersecrets/register.go

```diff
@@ -42,12 +41,8 @@ func NewUserSecretsManager(context facade.ModelContext) (*UserSecretsManager, er
 	}
 
 	return &UserSecretsManager{
-		authorizer:          context.Auth(),
-		resources:           context.Resources(),
-		authTag:             context.Auth().GetAuthTag(),
-		controllerUUID:      context.State().ControllerUUID(),
-		modelUUID:           context.State().ModelUUID(),
-		secretsState:        state.NewSecrets(context.State()),
-		backendConfigGetter: backendConfigGetter,
+		authorizer:    ctx.Auth(),
+		resources:     ctx.Resources(),
+		secretService: serviceFactory.Secret(backendConfigGetter),
```

Comment:
In the last PR, I was about to say that we should pass this into individual methods rather than the service.

Reply:
This will most likely be reworked once the backend service stuff fully lands. I just needed somewhere to put it until we get a bit further along.